docs/how_to_guides/configuring_metadata_stores/how_to_configure_an_expectation_store_in_amazon_s3.rst

```diff
@@ -3,16 +3,113 @@
+
+.. warning::
+
+    If you are also storing :ref:`Validations in S3, <how_to_guides__configuring_metadata_stores__how_to_configure_a_validation_result_store_in_s3>` please ensure that the ``prefix`` values are disjoint and one is not a substring of the other.
+
+    .. code-block:: yaml
+
+        expectations_store_name: expectations_S3_store
+
+        stores:
+            expectations_S3_store:
+                class_name: ExpectationsStore
+                store_backend:
+                    class_name: TupleS3StoreBackend
+                    bucket: '<your_s3_bucket_name>'
+                    prefix: '<your_s3_bucket_folder_name>'
+
+
+4. **Copy existing Expectation JSON files to the S3 bucket**. (This step is optional).
+
+    One way to copy Expectations into Amazon S3 is by using the ``aws s3 sync`` command.  As mentioned earlier, the ``base_directory`` is set to ``expectations/`` by default. In the example below, two Expectations, ``exp1`` and ``exp2`` are copied to Amazon S3.  Your output should looks something like this:
 
-If you want to be a real hero, we'd welcome a pull request. Please see :ref:`the Contributing tutorial <tutorials__contributing>` and :ref:`How to write a how to guide` to get started.
+    .. code-block:: bash
+
+        aws s3 sync '<base_directory>' s3://'<your_s3_bucket_name>'/'<your_s3_bucket_folder_name>'
+
+        upload: ./exp1.json to s3://'<your_s3_bucket_name>'/'<your_s3_bucket_folder_name>'/exp1.json
+        upload: ./exp2.json to s3://'<your_s3_bucket_name>'/'<your_s3_bucket_folder_name>'/exp2.json
+
+
+5. **Confirm that the new Expectations store has been added by running** ``great_expectations store list``.
+
+    Notice the output contains two Expectation stores: the original ``expectations_store`` on the local filesystem and the ``expectations_S3_store`` we just configured.  This is ok, since Great Expectations will look for Expectations in the S3 bucket as long as we set the ``expectations_name`` variable to ``expectations_S3_store``.
+
+    .. code-block:: bash
+
+        great_expectations store list
+
+        - name: expectations_store
+        class_name: ExpectationsStore
+        store_backend:
+            class_name: TupleFilesystemStoreBackend
+            base_directory: expectations/
+
+        - name: expectations_S3_store
+        class_name: ExpectationsStore
+        store_backend:
+            class_name: TupleS3StoreBackend
+            bucket: '<your_s3_bucket_name>'
+            prefix: '<your_s3_bucket_folder_name>'
+
+
+6. **Confirm that Expectations can be accessed from Amazon S3 by running** ``great_expectations suite list``.
+
+    If you followed Step 4, The output should include the 2 Expectations we copied to Amazon S3: ``exp1`` and ``exp2``.
```

Comment:
The other possibility is that the suite list is going to be empty. Do we want to add that?

Reply:
Yeah, worth a quick mention. "If you did not copy Expectations to the new Store, you will see..."

Reply:
done